packages/redux/src/checkout/selectors.ts

```diff
@@ -80,7 +80,7 @@ export const getCheckoutId = (state: StoreState): State['id'] =>
 export const getCheckout = (state: StoreState): CheckoutEntity | undefined => {
   const checkoutId = getCheckoutId(state);
 
-  return checkoutId && getEntityById(state, 'checkout', checkoutId);
+  return checkoutId ? getEntityById(state, 'checkout', checkoutId) : undefined;
```

Comment:
If I just use leave the selector I get an error on checkoutId because getCheckoutId either returns a number or null, and getEntityById function expects a number or string.
Maybe that's why there was a check for checkoutId before the selector, but even that had to be changed since it returned more types than expected by the function.

Since this was not my area I didn't want to mess with the rest of the functions.